features/edition-attachments.feature

```diff
@@ -13,33 +13,12 @@ Feature: Managing attachments on editions
     And I add an external attachment with the title "Beard Length Illustrations" and the URL "http://www.beardlengths.gov.uk"
     Then the publication "Standard Beard Lengths" should have 3 attachments
     When I set the order of attachments to:
```

Comment:
How is this passing for the Design System once when it's on a different page?

Reply:
I think it's passing right now because the Attachments Index page is still Bootstrap under the 'Preview next release' flag. So this feature spec is actually testing the Bootstrap functionality in both cases.

This will be something we'll need to tackle once the Attachments Index page gets tagged as `next_release: true`. I'm thinking we may need to adopt tags for features that _only_ run in Bootstrap or Design System modes rather than both – with this being the first example, due to it being a change of workflow.

Reply:
oh right ofc 🤦 yea all good then

Reply:
> oh right ofc 🤦

It took me some time to figure this out, too. So no shame.